manifests/server/config.pp

```diff
@@ -91,7 +91,7 @@
         quoted   => 'double',
       }
 
-      if ($::openldap::server::ldapi_ifs) {
+      if (!empty($::openldap::server::ldapi_ifs)) {
         shellvar { 'slapd_sockets':
           ensure   => present,
```

Comment:
Maybe we can remove the wrapping `if` and replace this with a conditional here:

```suggestion
          ensure   => bool2str(empty($openldap::server::ldapi_ifs), 'absent', 'present'),
```

This way, when changing the `ldapi_ifs` parameter from an array containing values to an empty array, the corresponding setting is removed, which is better than a transition from managed to unmanaged :-).

Reply:
It's early in the morning… so I guess that setting a variable in a condition and using this variable after is more readable…

Reply:
Good catch @smortex, change committed

Reply:
That look great! :coffee: